src/reactapp/src/pages/Login.js

```diff
@@ -1,67 +1,110 @@
-import React from 'react';
 import {
-    Flex,
-    Box,
-    FormControl,
-    FormLabel,
-    Input,
-    Checkbox,
-    Stack,
-    Link,
-    Button,
-    Heading,
-    Text,
-    useColorModeValue,
-} from '@chakra-ui/react';
+  Flex,
```

Comment:
One of the new .editorconfig settings above is to apply two spaces as the indent size on `.js` files, because these files tend to have deep indenting. Thus this file was reformatted from four to two spaces for each indent.